client/state/preferences/actions.js

```diff
@@ -28,26 +28,35 @@ export function fetchPreferences() {
 }
```

Comment:
While we should strive for code that is almost self-documenting. Comments can provide a great deal of context that a function signature itself can't provide. Usually the audience is for folks unfamiliar with the code, or mostly future me when I forget how something works.

So maybe something like:
```javascript
/**
 * Returns an action object that is used to signal storing a user preference for the _current_ page load. 
 * This is not to be confused with the `savePreference` action which will eventually store these values 
 * on the setting endpoint.
 * @param   { String | Number }               User preference key
 * @param   { String | Number | Object }      User preference value
 * @returns { Object }                        Action object
 */
export const setPreference = ( key, value ) => ( {
```